packages/client/src/workflow-options.ts

```diff
@@ -37,6 +34,36 @@ export interface WorkflowOptions extends CommonWorkflowOptions {
    * @default true
    */
   followRuns?: boolean;
+
+  /**
+   * Amount of time to wait before starting the workflow.
+   * This is ignored within a Schedule, as expected by {@link ScheduleClient.create}.
```

Comment:
You can remove that second sentence entirely. `ScheduleOptionsStartWorkflowAction` already pick only pertinent props from `WorkflowStartOptions`. Just make sure that this prop is effectively absent from `ScheduleOptionsStartWorkflowAction` (should be, but in case...)

Reply:
Yes, it was just that the programmer maybe confused if they set it, and they assume the schedule will also be delayed... 
Anyway, maybe more confusing than helpful, deleting...